packages/client/components/ReflectionCard/ReactjiCount.tsx

```diff
@@ -62,7 +62,8 @@ const ReactjiCount = (props: Props) => {
   const {onToggle, reactji, status, onTransitionEnd} = props
   if (!reactji) return null
   const {count, id, isViewerReactji} = reactji
-  const [, name = ''] = id.split(':')
+  const splitIndex = id.lastIndexOf(':')
+  const name = splitIndex !== -1 ? id.slice(splitIndex + 1) : ''
```

Comment:
out of curiosity, what's the reason for this change?

Reply:
Previously, the `id` would be of the form `<generated UUID>:<name>` so the name is always at index 1 after splitting, but with the team prompt response ID changes, the `id` is of the form `teamPromptResponse:<number>:<name>` so it's instead the last element after splitting, or the part of the string after the last index of `:`

Reply:
+1 can we refactor this to use the gqlId helper pattern? https://github.com/ParabolInc/parabol/blob/2708b9553cc212d6da9be951c6b486e46f57b68d/packages/client/shared/gqlIds/JiraProjectId.ts

That way future devs won't have to know how an ID is composed, they can just call `reactjiId.split(id)`

Reply:
Done